CHANGELOG.md

```diff
@@ -1,8 +1,187 @@
 # Changelog
 
+> For full v7 changelog, please refer to the [v.7x branch](https://github.com/mui/mui-x/blob/v7.x/CHANGELOG.md).
+
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 8.0.0-alpha.0
+
+<img width="100%" alt="MUI X v8 Alpha is live" src="https://github.com/user-attachments/assets/114cf615-b617-435f-8499-76ac3c26c57b">
+
+_Nov 7, 2024_
+
+We'd like to offer a big thanks to the 18 contributors who made this release possible. Here are some highlights ✨:
+
+- 🔁 Support [automatic parents and children selection](https://next.mui.com/x/react-tree-view/rich-tree-view/selection/#automatic-parents-and-children-selection) for the Rich Tree View components.
+
+  Special thanks go out to the community contributors who have helped make this release possible:
+  @belkocik, @GeorgiosDrivas, @k-rajat19, @kalyan90.
+  Following are all team members who have contributed to this release:
+  @alexfauquette, @arminmeh, @arthurbalduini, @cherniavskii, @flaviendelangle, @JCQuintas, @KenanYusuf, @LukasTy, @MBilalShafi, @michelengelen, @noraleonte, @oliviertassinari, @romgrk, @samuelsycamore.
+
+<!--/ HIGHLIGHT_ABOVE_SEPARATOR /-->
+
+### Data Grid
+
+#### `@mui/x-data-grid@8.0.0-alpha.0`
+
+- [DataGrid] Fix resizing right pinned column (#15107) @KenanYusuf
+- [DataGrid] Set default overlay height in flex parent layout (#15202) @cherniavskii
+- [DataGrid] refactor: baseMenuList & baseMenuItem (#15049) @romgrk
+- [DataGrid] refactor: remove more material imports (#15063) @romgrk
+- [DataGrid] - Fix grid overlay aligment with scroll for rtl (#15072) @kalyan90
+- [I10n] Improve Polish (pl-PL) locale (#15227) @belkocik
+
+#### `@mui/x-data-grid-pro@8.0.0-alpha.0` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
+
+Same changes as in `@mui/x-data-grid@8.0.0-alpha.0`, plus:
+
+- [DataGridPro] Fix column pinning layout (#14966) @cherniavskii
+
+#### `@mui/x-data-grid-premium@8.0.0-alpha.0` [![premium](https://mui.com/r/x-premium-svg)](https://mui.com/r/x-premium-svg-link 'Premium plan')
+
+Same changes as in `@mui/x-data-grid-pro@8.0.0-alpha.0`, plus:
+
+- [DataGridPremium] Server-side data source with row grouping (#13826) @MBilalShafi
+
+### Date and Time Pickers
+
+#### Breaking changes
+
+- The default DOM structure of the field has changed. [Migration Guide - New DOM structure for the field](https://github.com/mui/mui-x/blob/master/docs/data/migration/migration-pickers-v7/migration-pickers-v7.md#new-dom-structure-for-the-field)
```

Comment:
My issue with that is that it gets too big for no added value. Add it to each of the migration url and you have a lot of useless info.

It would be fine in the docs or in a blog post though

Reply:
I used the gh file because it is more static, the `next.mui.com` will make this url useless very soon

Reply:
> > I used the gh file because it is more static, the next.mui.com will make this url useless very soon
> 
> Not sure to follow you here

Wouldn't `next.mui.com` be "stuck" after we release v8 until we release v9 next year? Or is `next` always up-to-date? 🤔

Reply:
> Wouldn't next.mui.com be "stuck" after we release v8 until we release v9 next year? Or is next always up-to-date? 🤔

AFAIK, we will replace all the `next.mui` links with `mui.com` links when releasing major.

Reply:
I confirm all links to the docs that appears somewhere else should be prefixed with `next.mui.com` and updated between version.

We had the same issue with JSDocs and agreed to use `https://mui.com` and replace them by `https://next.mui.com` according to the current status of the branch

Reply:
Updated